pkgs/tools/backup/mastodon-archive/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, python
```

Comment:
```suggestion
, python3
```
otherwise it's harder to tell from this file what `python` actually is.

Reply:
Done